src/mcts/search.cc

```diff
@@ -858,6 +863,18 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
         return NodeToProcess::Extension(node, depth);
       }
     }
+    Node* possible_shortcut_child = node->GetCachedBestChild();
+    if (possible_shortcut_child) {
+      // Add two here to reverse the conservatism that goes into calculating the
+      // remaining cache visits.
+      collision_limit =
+          std::min(collision_limit, node->GetRemainingCacheVisits() + 2);
```

Comment:
I'm fine also with `+/- 1` instead of +2 and even with +0.

Reply:
Maybe a follow up where someone does some tuning to decide which gets best elo.